app/views/request_mailer/very_overdue_alert.text.erb

```diff
@@ -1,9 +1,15 @@
 <%= raw @info_request.public_body.name %> <%= _('are long overdue.')%>
 
+<% if @info_request.public_body.not_subject_to_law? %>
```

Comment:
As per the comment on `app/views/request_mailer/overdue_alert.text.erb`. I'd remove the conditional and use "as _normally_ required by law".